packages/create-astro/test/utils.js

```diff
@@ -37,16 +37,24 @@ export function promiseWithTimeout(testFn) {
 export const PROMPT_MESSAGES = {
 	directory: 'Where would you like to create your new project?',
 	template: 'How would you like to setup your new project?',
+	packageSucceed: 'Sounds good! The NPM package name was updated',
+	packageIgnored: 'You have to update the NPM package name manually.',
 	typescript: 'How would you like to setup TypeScript?',
 	typescriptSucceed: 'next',
 };
 
 export function setup(args = []) {
-	const { stdout, stdin } = execa('../create-astro.mjs', [...args, '--skip-houston', '--dryrun'], {
+	const { stdout, stdin } = execa('../create-astro.mjs', [...args, '--skip-houston', '--dry-run'], {
 		cwd: testDir,
 	});
 	return {
 		stdin,
 		stdout,
 	};
 }
+
+export async function getNPMVersion() {
+	const { stdout: npmVersion } = await execa('npm', ['-v']);
+	const [major, minor, patch] = npmVersion.split('.').map(Number);
+	return { major, minor, patch }
+}
```

Comment:
This is unused AFAICT

Reply:
@natemoo-re `getNPMVersion()` is used by `package-step.test.js`.

I need this function because the sub command `npm pkg` is not supported in NPM v6 and older. Currently, I don't find out anyway to implement test case that run on specific Node version except using `if-else` condition.

I will move it into `package-step.test.js` since it will be used in that test file at this moment.